apps/web/app/explore/page.tsx

```diff
@@ -39,6 +39,7 @@ async function Explore() {
             </div>
           );
         })}
+        <div className="sticky w-full h-[15vh] bottom-0 bg-gradient-to-t from-[#000] drop-shadow-[0_4px_4px_0_rgba(0,0,0,0.25)]"></div>
```

Comment:
change the `from-[#000]` to  ->  `from-[#262626]` . Other are perfect

Reply:
Changed